cmake/templates/OpenCVConfig-CUDA.cmake.in

```diff
@@ -10,10 +12,10 @@ set(OpenCV_CUDNN_VERSION    "@CUDNN_VERSION@")
 set(OpenCV_USE_CUDNN        "@HAVE_CUDNN@")
 
 if(NOT CUDA_FOUND)
-  find_host_package(CUDA ${OpenCV_CUDA_VERSION} EXACT REQUIRED)
+  find_host_package(CUDA ${OpenCV_CUDA_VERSION_MIN} EXACT REQUIRED)
```

Comment:
From CMake documentation:
> The EXACT option requests that the version be matched exactly. This option is incompatible with the specification of a version range.

The option is redundant for the case with softer case, if minor version is included.

Reply:
as I tested: `find_host_package(CUDA 12 EXACT REQUIRED)` find whatever 12.x, it does not care the minor
`find_host_package(CUDA 12.6 EXACT REQUIRED)` find whatever 12.6 it does not care the patch
the newly introduced `get_version_range` does not give you minor version if ignored, and give you the exact minor version the CUDA_VERSION_STRING give out.